.gitignore

```diff
@@ -39,6 +39,5 @@ luac.out
 *.x86_64
 *.hex
 
-plenary.nvim/
-nvim-treesitter/
-mini.nvim/
+# Directory containing testing dependencies downloaded during test run
+.min/
```

Comment:
Just curious, why `.min` ? Is it some kind of pattern?
I'd go for `.vendor` or `.deps` or something like that.

Reply:
`.min` because it's for the minimal init 🤷. First thing that came to mind, I'll change it over to `.deps` I guess.

Reply:
Done

Reply:
You updated the code to point to `.deps`, but gitignore and git workflow still points to `.min`.